playbooks/roles/epel_repositories/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+release: "{{ ansible_distribution_version.split('.')[0] }}"
```

Comment:
Ah, this is literally a copy and paste from where `release` is used elsewhere.  We really don't need the variable then.